source/MBF.SAMD51.SystemCore.pas

```diff
@@ -57,7 +86,11 @@   TSAMD51SystemCore = record helper for TSystemCore
 var
   SystemCore : TSystemCore;
   //This var is needed to communicate CPU Speed to FreeRTOS
+  {$ifdef freertos_fat}
```

Comment:
Why was the extra 'external needed?

Reply:
My FreeRTOS config has its own SystemCoreClock.
https://github.com/LongDirtyAnimAlf/mbf-freertos-wio/blob/master/SamplesBoardSpecific/WioTerminal/Hardware/ArduinoCore/libraries/Seeed_Arduino_FreeRTOS/src/boards/SAMD51/FreeRTOSConfig.h#L4
Might not be necessary anymore. Will look into.